src/goto-instrument/cover_basic_blocks.cpp

```diff
@@ -87,22 +87,22 @@ cover_basic_blockst::cover_basic_blockst(const goto_programt &_goto_program)
     update_covered_lines(block_info);
 }
 
-unsigned cover_basic_blockst::block_of(goto_programt::const_targett t) const
+std::size_t cover_basic_blockst::block_of(goto_programt::const_targett t) const
```

Comment:
Why all these changes to a `std::size_t` when the map itself (`block_mapt`) still maps to `unsigned`? And the commit message hasn't got a single word about this?!

Reply:
This was to fit the function type of the interface. I guess I will add a new commit at the beginning to do these type changes.